app/connectors/PenaltiesConnector.scala

```diff
@@ -36,4 +37,9 @@ class PenaltiesConnector @Inject()(httpClient: HttpClient,
   def getPenaltiesData(enrolmentKey: String)(implicit user: User[_], hc: HeaderCarrier): Future[ETMPPayload] = {
     httpClient.GET[ETMPPayload](s"$penaltiesBaseUrl${getPenaltiesDataUrl(enrolmentKey)}")
```

Comment:
Can you add the new query parameter to this call? as without it calls for the old model will return a 400 bad request